### ISSUE TITLE
Feature Request: Flag to Create `removed` blocks

### ISSUE THREAD

**Issue by philnielsen-mercury** [kishaningithub/tf-import-gen#37]
Hi! Sweet tool, I've been using this a bunch! Any chance you would be open to a contribution (or would be interested in adding) a flag that generates these as `removed` blocks for the resources instead of imports?
```hcl
removed {
  from = module.example_bucket.this
  lifecycle {
    destroy = false
  }
}
```

**Comment by kishaningithub** [comment]
Thanks! Never knew about the existence of the `removed` block before..

Can you explain your use case a bit more so that I can understand better?

**Comment by philnielsen-mercury** [comment]
They are [pretty new](https://support.hashicorp.com/hc/en-us/articles/34185721057555-Removing-a-Resource-from-the-Terraform-State-Using-the-removed-block) in tf 1.7! 


### Current Workflow
We are using your tool to migrate resources from one root module to another, so the workflow is as follows:
1. Generate the import blocks for the module calls or resources we want to move
```bash
terraform show -json | tf-import-gen module.example
```
output:
```hcl
import {
  to = module.example.aws_s3_bucket.this
  id = <EXAMPLE ARN>
}
```
2. move all the resources to the new module by moving the files
3. generate removed blocks in the old location so that old root module forgets about the resources we don't want it managing anymore


### Desired
Currently we are using GPT 4 for this final step which sorta gets us most of the way there. I could probably get some sed/awk bash script going here, but since this tool already has a lot of the bones I started looking to see if we could generate the removed blocks with it!
```bash
terraform show -json | tf-import-gen -r module.example
```

expected output
```hcl
removed {
  from = module.example.aws_s3_bucket.this
  lifecycle {
    destroy = false
  }
}
```

There are some minor quirks with removed blocks, like it doesn't support [instances of resources](https://github.com/hashicorp/terraform/issues/34439#issuecomment-2178128452), so I didn't get too into it yet and wanted to see if this was something you would be interested in adding!

**Comment by kishaningithub** [comment]
@philnielsen-mercury Apologise for the late response and thanks for the detailed explanation !

I am definitely interested to explore further on this. One curious question, For the movement of resources from one module to another you considered [moved blocks](https://developer.hashicorp.com/terraform/language/moved) ?

**Comment by philnielsen-mercury** [comment]
yes! we are moving between two modules that are in different workspaces in terraform cloud so moved blocks unfortunately don't work here.

**Comment by kishaningithub** [comment]
I am definitely up for getting this done. If you want to work on this also let me know will help you out the best i can

**Comment by philnielsen-mercury** [comment]
hey @kishaningithub , I don't currently have cycles for this but possibly in a few weeks! (but I'm not currently actively working on it so if you want to take a stab at it go ahead)
